release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -145,43 +185,210 @@ def crop_and_flip_image(row):
     return row
 
 
+def decode_image_crop_and_flip(row):
```

Comment:
Can we import these from image_loader_microbenchmark instead? Or put them all in the same util file.